File-Formats/PDF-to-image/Converting-PDF-pages-into-Images.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Converting PDF pages into images| Syncfusion
+description: Learn about Converting PDF pages into images using Syncfusion PdfToImageConverter.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+
+# Converting PDF pages into images
+
+PdfToImageConverter allows exporting pages of a PDF file into JPG, PNG, TIFF, and BMP formats using Convert methods. This option helps to convert PDF pages into images.
```

Comment:
PdfToImageConverter allows exporting pages of a PDF document into the stream using the Convert method.

Reply:
I have modified as mentioned